### DISCUSSION TITLE
Is it possible to open a specific iTerm saved window arrangement?

### DISCUSSION THREAD

**Discussion by lsps9150414** [ttscoff/bunch-docs#215] [Q&A] [answered]
I have several iTerm window arrangements. I would like a bunch to open up a specific one. Is this currently possible?

**Comment by ttscoff** [comment, marked as answer]
I believe you can assign keyboard shortcuts to window arrangements, right? Then you could have Bunch send the keyboard shortcut directly to iTerm when launching...
